### PR TITLE
[tests-only][full-ci]add scenario for search folder tag

### PR DESCRIPTION
**Pull request by nabim777**
## Description
This PR adds the API tests for full search api. The scenarios added in this PR are
- search folder using tag

## Related Issue
- https://github.com/owncloud/ocis/issues/6606#event-9635806756

## Motivation and Context
- there was no test coverage for api test for search folder using tag. so, this PR covers the require scenario test case

## How Has This Been Tested?
- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 1]
```diff
@@ -29,6 +29,27 @@ Feature: full text search
...
+    And user "Alice" has created folder "uploadFolder1"
+    And user "Alice" has created folder "uploadFolder2"
+    And user "Alice" has created folder "uploadFolder3"
+    And user "Alice" has created the following tags for folder "uploadFolder1" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for folder "uploadFolder2" of the space "Personal":
+      | tag1 |
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
```
Comment (grgprarup) [inline tests/acceptance/features/apiGraph/fullSearch.feature:43]:
```suggestion
    And the search result of user "Alice" should contain only these folders:
```
OR
   ` And the search result of user "Alice" should contain only these resources:`

Comment (nabim777) [reply tests/acceptance/features/apiGraph/fullSearch.feature]: okay

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 2]
```diff
@@ -29,6 +29,27 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search folder using a tag
```
Comment (grgprarup) [inline tests/acceptance/features/apiGraph/fullSearch.feature:32]:
```suggestion
  Scenario Outline: search folders using a tag
```

**Review by saw-jan — COMMENTED** [review]
others LGTM

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]

**Review by grgprarup — APPROVED** [review]
LGTM :+1:

**Pull request merged**